src/Agent.Listener/Configuration/RSACngKeyManager.cs

```diff
@@ -0,0 +1,108 @@
+﻿#if OS_WINDOWS
```

Comment:
Consider: Is #if def required on the two key manager implementations? If not, additional compile time checking  on all OS's would be gained by not doing #if def.